front-script/public/index.html

```diff
@@ -12,5 +12,6 @@
 				<!-- the chat will be generated inside this -->
 		</div>
 	</div>
+	<script src="https://www.gstatic.com/firebasejs/5.8.4/firebase.js"></script>
```

Comment:
this index.html file doesn't belong in the frontscript release it represents the clients web page ... it's for testing purpose